[RUM] add documentation for loading time and loading type (#7680)
DataDog/documentation: content/en/real_user_monitoring/data_collected/_index.md

| `network.client.geoip.country.iso_code`     | string | [ISO Code][3] of the country (example: `US` for the United States, `FR` for France)                                                  |
| `network.client.geoip.continent.code`       | string | ISO code of the continent (`EU`, `AS`, `NA`, `AF`, `AN`, `SA`, `OC`)                                                                 |
| `network.client.geoip.continent.name`       | string | Name of the continent (`Europe`, `Australia`, `North America`, `Africa`, `Antartica`, `South America`, `Oceania`)                    |
| `network.client.geoip.subdivision.name`     | string | Name of the first subdivision level of the country (example: `California` in the United States or the `Sarthe` department in France) |
| `network.client.geoip.subdivision.iso_code` | string | [ISO Code][3] of the first subdivision level of the country (example: `CA` in the United States or the `SA` department in France)    |
| `network.client.geoip.city.name`            | string | The name of the city (example `Paris`, `New York`)                                                                                   |

## Extra Attribute

> In addition to default attributes, you can add your [specific global context][4] to all events collected. This provides you the ability to analyze the data for a subset of users: group errors by user email, understand which customers have the worst performance, etc.

Reviewer: ```suggestion
In addition to default attributes, add [specific global context][4] to all events collected. This provides the ability to analyze the data for a subset of users. For example, group errors by user email, or understand which customers have the worst performance.
```
